test/components/categoryAxisTests.ts

```diff
@@ -108,4 +108,22 @@ describe("Category Axes", () => {
 
     svg.remove();
   });
+
+  it("layout calculated correctly under 0 width/height surrounding component conditions", () => {
```

Comment:
Is there a test that doesn't involve `Table`? What was the root cause of the bug?